lib/themes/dosomething/paraneue_dosomething/js/images/ImageCrop.js

```diff
@@ -322,8 +322,7 @@ define(function(require) {
     var $cropHeight   = $form.find("input[name='crop_height']");
     var $cropRotation = $form.find("input[name='crop_rotate']");
     var $captionField = $form.find("input[name='caption']");
-    var caption       = $("#dosomething-reportback-image-form").find("#caption").val();
-
+    var caption       = $("#dosomething-reportback-image-form").find("input[name='modal-caption']").val();
```

Comment:
Not crucial but future update, the caption input has an id of `#modal-caption` which you could use in the `find()` instead :)